examples/platforms/efr32mg1/crypto/efr32-mbedtls-config.h

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
LGTM :+1:

You could add `MBEDTLS_SHA1_ALT` to also accelerate SHA-1 operations in case the user application decides to rely on SHA-1. Though SHA-1 is no longer trusted in cryptographic contexts given recent collision attacks, it still has its use for generic checksumming.

Reply:
Let's leave it off.  A customer can add it if needed/wanted.